Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
     package="de.rki.coronawarnapp"
     tools:ignore="LockedOrientationActivity">
 
-    <uses-sdk tools:overrideLibrary="com.google.zxing.client.android" />
+    <uses-sdk tools:overrideLibrary="com.google.zxing.client.android,androidx.security" />
```

Comment:
I agree specially that official API 21 support from Google still in alpha https://developer.android.com/jetpack/androidx/releases/security#security-crypto-1.1.0-alpha01.
I would love to test that on real Android 5 devices as well as ENF

Reply:
Android 5 support is not available in the version that you are using, as you are using `1.0.0-rc03`. It would be an option to update to this version, but I wouldn't consider it to be the best one for your project.

Reply:
Unfortunately we won't be able to use alpha versions here,  I would suggest to revert this change and keep our minSDK = 23 and we can merge this PR. In our side this would not be a problem as we still guard our code to run on API 23+.

By the changes in here we have increased the compatibility with pre Android M versions. We can lower our minSDK when all blockers are addressed. @d4rken what do think?

Reply:
I added a change like @d4rken suggested: f5a137cd9ea445e4bbcb6e761eb6f2a7860b17e3.

This appears to be the only place from which `EncryptedPreferencesMigration` can be called, so the code from the `androidx.security` library that doesn't work with < 23 won't be invoked on those devices.

Reply:
Thanks, looks good.

Reply:
Thanks you for addressing this.